atmospheric_transmittance.py

```diff
@@ -0,0 +1,194 @@
+import os
+import json
+import requests
+
+import astropy.units as u
+from astropy.io import fits
+
+
+def get(airmass=1.0, pwv_mode='pwv', season=0, time=0, pwv=3.5, msolflux=130.0,
```

Comment:
Could you give a more specific function name than `get`? Perhaps `get_atmospheric_transmittance`?